plone-5.2.x.cfg

```diff
@@ -23,6 +23,7 @@ plone-user = admin:admin
 zc.recipe.cmmi = 1.3.6
 
 [versions:python27]
+check-manifest = 0.41
```

Comment:
Didn't you want to pin this on `buildout.coredev` because it is not used there?

Reply:
It *is* pinned in coredev, only not in `versions.cfg` but in `tests.cfg`, as it is not used by core Plone. But we want to [merge the versions](https://github.com/plone/buildout.coredev/pull/686) from those two files at some point.

Reply:
It might be good to put a FIXME comment with this information.